src/materialized/src/bin/materialized/main.rs

```diff
@@ -575,7 +575,8 @@ os: {os}
 cpus: {ncpus_logical} logical, {ncpus_physical} physical, {ncpus_useful} useful
 cpu0: {cpu0}
 memory: {memory_total}KB total, {memory_used}KB used{memory_limit}
-swap: {swap_total}KB total, {swap_used}KB used{swap_limit}",
+swap: {swap_total}KB total, {swap_used}KB used{swap_limit}
+Timely workers: {workers}",
```

Comment:
nit: this should be "dataflow workers" or "workers" to match how we talk about the `--workers` param in other places, e.g. https://github.com/MaterializeInc/materialize/blob/d4a3fe719b50892d341d942692e3772ae43a21c1/src/materialized/src/bin/materialized/main.rs#L148-L150

Reply:
```suggestion
dataflow workers: {workers}",
```